drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/pom.xml

```diff
@@ -136,6 +136,16 @@
       <artifactId>jackson-databind</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-core</artifactId>
+    </dependency>
+
+    <dependency>
```

Comment:
@danielezonca 
I know it is unavoidable, but did you take a look at relationship/binding this dependency creates with other jars? I mean, with jars taken in by kie-dmn-core/kie-dmn-model

Reply:
Yes this dependency binds dmn modules and scenario more that _expected_ but it cannot be avoided. Maybe in the future can be removed or at least limited to `test` scope